src/agent/samples/policy/yaml/pod/pod-ubuntu.yaml

```diff
@@ -13,11 +13,11 @@ metadata:
 spec:
   containers:
     - name: ubuntu-main
-      image: "docker.io/library/ubuntu:latest"
+      image: "docker.io/library/ubuntu:jammy-20240212"
```

Comment:
Start using fixed tags instead of `latest` on common images that sometimes fail in our internal e2e tests due to "outdated yaml".
- Pro: increase test stability, should reduce/eliminate test failures due to "outdated yaml"

- Con: We are no longer testing `latest` and now we should move this fixed tags forward from time to time

I'm leaning on this change because we get a sporadic test failure every week roughly, but there's only been one instance that it has helped us to test the latest version of an image.